ui/app/src/components/shared/ResourceHistoryListItem.tsx

```diff
@@ -14,15 +16,31 @@ export const ResourceHistoryListItem: React.FunctionComponent<
       color: DefaultPalette.neutralSecondary,
     },
   };
+  const [isErrorPanelOpen, setIsErrorPanelOpen] = React.useState(false);
+  const isError = typeof props.val === "string" && (props.val.includes("Error:") || props.val.includes("error:"));
+
   return (
     <>
       <Stack wrap horizontal>
         <Stack.Item styles={stackItemStyles} style={{ width: "20%" }}>
           {props.header}
         </Stack.Item>
-        <Stack.Item styles={stackItemStyles} style={{ width: "80%" }}>
-          : {props.val}
-        </Stack.Item>
+        {isError ? (
+          <>
+            <Stack.Item styles={stackItemStyles} style={{ width: "80%" }}>
+              <Link onClick={() => setIsErrorPanelOpen(true)}>An error occurred, click to view the error details</Link>
```

Comment:
nit: Can we make this two sentences, or use a semi-colon please?
```suggestion
              <Link onClick={() => setIsErrorPanelOpen(true)}>An error occurred; click to view the error details</Link>
```